static/js/message_edit.js

```diff
@@ -474,6 +475,8 @@ function edit_message($row, raw_content) {
     const $message_edit_countdown_timer = $row.find(".message_edit_countdown_timer");
     const $copy_message = $row.find(".copy_message");
 
+    $message_edit_topic_propagate.toggle(!is_content_editable);
```

Comment:
Can you explain this line? It seems to me that in situations where both the content and the topic are editable, that we'd incorrectly not display the menu.

Reply:
@timabbott you are right, if the content is editable, the menu is not immediately displayed. But this is the expected behavior, since the main goal of editing a message, when the content is still editable, is expected to be editing the content. And because of this it would be confusing to immediately show the menu then, as the user might not know what this refers to.
(See also my previous messages with @alya or the issue description)

Reply:
Could you please add a code comment before this line explaining it? Thanks!

Reply:
@alya alright, added the comment

Reply:
So I'm glad we spent some extra attention on this line, because it's wrong in the case that the user can't edit the message at all (aka "View source"):

![image](https://user-images.githubusercontent.com/2746074/179319205-5d964d85-033d-4fc3-9f53-6769f5e0150f.png)

I'll fix this before merging, since I think the fix is straightforward.